applications/smart-distancing/eval/README.md

```diff
@@ -0,0 +1,52 @@
+# Smart Social Distancing - Performance Metrics for Object Detection
+
+This module provides easy-to-use functions which enables you to evaluate available exported results from detectors.
+
+### How to use
+**Step 1: Build Image**
+```
+cd neuralet/applications/smart-distancing/eval
+# Build Docker image
+docker build -f Dockerfile-eval -t detector-eval .
+```
+**Step 2: Create The Input Files**
+
+* Create a separate ground truth text file for each image in the directory `neuralet/applications/smart-distancing/eval/eval_files/groundtruths/`.
+* In these files each line should be in the format: `<class_name> <left> <top> <width> <height>`.
```

Comment:
please mention other options, for example normalized coordinates or <left> <top> <right><bottom> format

Reply:
The module will generate coordinates on an unnormalized format automatically, and there is no need to use other options at the code. So I didn't mention other arguments.

Reply:
For the sake of simplicity, I just used 3 arguments. The evaluatory python module has various options which can confuse users. At the docker file, the required arguments are used for the evaluator module.